mutalyzer/GenRecord.py

```diff
@@ -312,6 +312,21 @@ def findGene(self, name) :
         return None
     #findGene
 
+    def getInfoByTranscriptID(self, transcript_id):
```

Comment:
So *name* would be equivalent to what you called *selector* above?

I've changed it to selector in the code, as name is too generic and confusing in this particular function.